src/clients/dotnet/TigerBeetle.Tests/IntegrationTests.cs

```diff
@@ -1109,19 +1111,19 @@ private static void AssertTransfer(Transfer a, Transfer b)
 
     internal class TBServer : IDisposable
     {
-        public const string TB_PORT = "3001";
-
         // Path relative from /TigerBeetle.Test/bin/<framework>/<release>/<platform> :
         private const string PROJECT_ROOT = "../../../../..";
         private const string TB_PATH = PROJECT_ROOT + "/../../../zig-out/bin";
         private const string TB_EXE = "tigerbeetle";
         private const string TB_FILE = "dotnet-tests.tigerbeetle";
         private const string TB_SERVER = TB_PATH + "/" + TB_EXE;
         private const string FORMAT = $"format --cluster=0 --replica=0 --replica-count=1 ./" + TB_FILE;
-        private const string START = $"start --addresses=" + TB_PORT + " --cache-grid=256MB ./" + TB_FILE;
+        private const string START = $"start --addresses=0  --cache-grid=256MB ./" + TB_FILE;
 
         private readonly Process process;
 
+        public readonly string address;

```

Comment:
```suggestion
        public string Address { get; }
```

Idiomatic way of publishing a read-only field is via property 🤷‍♂️, just replace `server.address` with `server.Address`.